### ISSUE TITLE
Add a option to disable pop-ups( In new tab) for selected websites

### ISSUE THREAD

**Issue by BigDogg99** [bromite/bromite#2213]
### Preliminary checklist

- [X] I have read the [README](https://github.com/bromite/bromite/blob/master/README.md)
- [X] I have read the [FAQs](https://github.com/bromite/bromite/blob/master/FAQ.md).
- [X] I have searched [existing issues](https://github.com/bromite/bromite/issues) for my feature request. This is a new issue (NOT a duplicate) and is not related to another issue.
- [X] This is a feature request for the Bromite browser; not the website nor F-Droid nor anything else.

### Is your feature request related to privacy?

Yes

### Is there a patch available for this feature somewhere?

no

### Describe the solution you would like

Some persistant ads and malware sites open in new tab by clicking anyware on some [sites](https://9anime.id/watch/lupin-the-3rd-part-6.oj9o4/ep-14) there should  is a option to disable any pop ups from selected sites 

### Describe alternatives you have considered

Only other alternative is using ublock as default adblocker for this browser

**Comment by AxeldeWater** [comment]
There is a vanilla Chromium option to block popups from all sites in Setting > Site settings > Pop-ups and Redirects. Is the problem that this feature is not working properly or do you want Bromite to add the possibility to only block pop-ups on a selection of sites and allow pop-ups for all other sites?

**Comment by pritkr** [comment]
Chromium implementation of pop-up is not effective mainly to have compatibilty with sites that use popups for useful use case. [ADM](https://play.google.com/store/apps/details?id=com.dv.adm) for example has good popup blocking but it is disabled by default and breaks some sites. It can be tested on sites like https://gplinks.co/42Nxaos (not copyright friendly link) 
![Screenshot_2022-08-03-19-08-39-206_com dv adm](https://user-images.githubusercontent.com/81620727/182622813-4cc54ba1-2145-472b-9de1-daec6606e7b4.jpg)
![Screenshot_2022-08-03-19-08-19-081_com dv adm](https://user-images.githubusercontent.com/81620727/182623087-9f39eec6-4413-445b-a66f-42076eac87b4.jpg)

**Comment by csagan5** [comment]
I do not understand what is this feature request about and I am not going to open any of these links; please submit an example in HTML+JavaScript of the behavior that you think should be handled by the browser differently with current existing features.

**Comment by Disqu3-mirOir-qt** [comment]
![Screenshot_20220809-115212.jpg](https://user-images.githubusercontent.com/41585567/183619881-5e4bad61-8e8d-44cb-8f60-e4618dd180ff.jpg)


uqload player for example 

already blocked by adguard (personal rules filter)

i think can be solved by userscript too, It remains very technical and complicated for simple users

**Comment by AxeldeWater** [comment]
> 
> https://user-images.githubusercontent.com/109841554/183607421-b4768733-044d-4f36-9fd1-0a83a8e64aa4.mp4
> 
> 

Just off topic, I don't know if you use this site regularly or not, but if you know this site keeps downloading random shit, even though it gets blocked by an adblocker or on some other browser... Don't fucking use it... Just go to theindex.moe if you want an anime site that does not try to download viruses on your device? Just in case you use that site often, personal tip.
